.github/workflows/build.yml

```diff
@@ -320,6 +333,18 @@ jobs:
         run:
           cp LICENSE ./control-plane
       - uses: hashicorp/actions-docker-build@v1
+
+      # This naming convention will be used ONLY for per-commit dev images
+      - name: Set docker dev tag
+        run: |
+          echo "full_dev_tag=${{ env.version }}"
+          echo "full_dev_tag=${{ env.version }}" >> $GITHUB_ENV
+          echo "minor_dev_tag=$(echo ${{ env.version }}| sed -E 's/([0-9]+\.[0-9]+)\.[0-9]+(-[0-9a-zA-Z\+\.]+)?$/\1\2/')" 
+          echo "minor_dev_tag=$(echo ${{ env.version }}| sed -E 's/([0-9]+\.[0-9]+)\.[0-9]+(-[0-9a-zA-Z\+\.]+)?$/\1\2/')" >> $GITHUB_ENV
+
+      - name: Docker Build (Action)
+        uses: hashicorp/actions-docker-build@v1
+        if: ${{ !matrix.fips }}
```

Comment:
I don't think it will make a difference in this case, but I don't think we need this `if` since we don't support FIPS in this version.